README.md

```diff
@@ -30,6 +30,7 @@ you might also find an early build of [python3.10] useful
 | YTT204 | `sys.version_info.minor` compared to integer (python4) |
 | YTT301 | `sys.version[0]` referenced (python10)                 |
 | YTT302 | `sys.version` compared to string (python10)            |
+| YTT303 | `sys.version[:1]` referenced (python310)               |
```

Comment:
python10

Reply:
I can write a code generator for this / copy one of the ones from pygrep-hooks or another repo

Reply:
ooh yeah that would be nice -- the one in `pygrep-hooks` is a pretty decent example

Reply:
Fixed manually for now